src/sas/sascalc/invariant/invariant.py

```diff
@@ -964,13 +979,17 @@ def get_surface_with_error(self, contrast, porod_const, extrapolation=None):
         from the contrast and porod_constant wich are currently user inputs
         with no option for any uncertainty so no uncertainty can be calculated.
         However we include the uncertainty computation for future use if and
-        when these values get an uncertainty. This is given as:
+        when these values get an uncertainty. This is given as: ::
+
+            ds = sqrt[(s\'_cp)**2 * dcp**2 + (s\'_contrast)**2 * dcontrast**2]
+
+        where s'_x is the partial derivative of S with respect to x
```

Comment:
Why the backslash in `s\'_cp` but not `s'_x`?

Do you need to escape the previous `'`? E.g., because there are two on the same line and RST is interpreting that as meaningful?

Reply:
Yes, the '…' on the equation line was being interpreted as a quotation. (Actually, it still might be now I look at it).

The literal block is implicitly Python code and get syntax highlighted. Perhaps for this particular docstring, something other than prime is needed to indicate the derivative. Any thoughts on what?

Reply:
`dS = sqrt[(dS/dcp)**2 * dcp**2 + (dS/dcontrast)**2 * dcontrast**2]`? 

Or just switch to math since right now it is neither math nor python:
```
\Delta S^2 = \left(\frac{\partial S}{\partial \mathrm{cp}}\right)^2 \Delta \mathrm{cp}^2 
+ \left(\frac{\partial S}{\partial \mathrm{contrast}}\right)^2 \Delta \mathrm{contrast}^2
```

Reply:
Or `.. code-block:: none` for minimal change.